app/views/report/_form_filter_chargeback.html.haml

```diff
@@ -1,4 +1,17 @@
 - url = url_for(:action => 'form_field_changed', :id => "#{@edit[:rpt_id] || 'new'}")
+- if @edit[:new][:model] == "ChargebackVm"
+  %h3
+    = _('Chargeback Resources')
+  .form-horizontal
+    .form-group
+      %label.control-label.col-md-2
+        = _('Include Capacity & Utilization Metrics')
+      .col-md-8
+        = check_box_tag("cb_include_metrics", true, @edit[:new][:cb_include_metrics].nil? || @edit[:new][:cb_include_metrics],
```

Comment:
Since you have the same condition when setting `@edit[:new][:cb_include_metrics]`, here you can just use that, right? (As in, it will never be `nil` here.)